src/js/Audit.js

```diff
@@ -22,8 +22,13 @@ goog.provide('axs.AuditConfiguration');
 /**
  * Object to hold configuration for an Audit run.
  * @constructor
+ * @param {?Object=} config Configuration object
```

Comment:
Can you add more detailed documentation for this, stating what can and cannot be configured this way?